File: a149(multiply)/Python/multiply.py
while True:
    try:
        num_of_cases = int(input())
        answers = []
        for i in range(num_of_cases):
            case = input()
            product = 1
            for n in case:
                product *= int(n)
            answers.append(product)
        for product in answers:
            print(product)
    except:
        break

File: a149(multiply)/C++/multiply.cpp
#include <iostream>

int main(int argc, char const *argv[]) {
  int num_of_cases;
  while (std::cin >> num_of_cases) {
    int answers[num_of_cases] = {0};
    for (int i = 0; i < num_of_cases; i++) {
      int product = 1;
      std::string s;
      std::cin >> s;
      for (char n: s) {
        product *= (int)n - 48;
      }
      answers[i] = product;
    }
    for (int product: answers) {
      std::cout << product << '\n';
    }
  }

  return 0;
}
